apps/rehabstod/src/components/Form/NumberInput.tsx

```diff
@@ -1,49 +1,36 @@
-import { IDSInput } from '@frontend/ids-react-ts'
-import { ChangeEvent } from 'react'
+import { ChangeEvent, useId } from 'react'
+import { classNames } from '../../utils/classNames'
 import { TooltipIcon } from '../TooltipIcon/TooltipIcon'
+import { Input } from './Input'
 
 export function NumberInput({
   label,
   onChange,
   onBlur,
   description,
-  id,
   value,
-  isRange,
   max,
   min,
-  className,
-  classNameInput,
-  novalidation,
+  error,
+  inline = false,
 }: {
   label: string
   onChange: (event: ChangeEvent<HTMLInputElement>) => void
   onBlur?: () => void
   description?: string
-  id: string
   value: number | string
-  isRange?: boolean
+  inline?: boolean
   max?: number | string
   min?: number | string
-  className?: string
-  classNameInput?: string
-  novalidation?: boolean
+  error?: boolean
 }) {
+  const id = useId()
   return (
-    <IDSInput className={`${isRange && 'flex items-baseline gap-3'} ${className}`} novalidation={novalidation}>
```

Comment:
`IDSInput` has currently multiple styling issues, border for error input is not rendered correctly. Full-width is not respected. Renamed "isRange" to "inline" to describe layout option.